python/lsst/ts/tunablelaser/component.py

```diff
@@ -121,9 +129,20 @@ async def set_output_energy_level(self, output_energy_level):
         self.log.debug(f"Changing output energy level={output_energy_level}")
         await self.m_cpu800.set_output_energy_level(output_energy_level)
 
+    async def set_trigger_mode(self):
+        """Set trigger mode for laser."""
```

Comment:
Please clarify the difference between this and set_burst_mode

Also please use a separate detailed state for these two modes (if they really are importantly different from a SAL user's perspective)

Reply:
What does this method do if the laser is not in burst mode? Does the user get an informative error? If not,  please fix that (if necessary, by checking the mode yourself before commanding the laser).

Reply:
These comments still apply and please delete the `start_trigger` method below.